chore: update ccip tests to use new test engine.

jkongie opened the pull request:
This commit updates the ccip tests to use the new test engine. This covers the simple conversions mainly revolving EVM and Aptos chains.

graham-chainlink commented on deployment/ccip/changeset/v1_6_2/cs_configure_usdc_token_pool_test.go:
```diff
@@ -129,53 +125,41 @@ func setupUSDCTokenPoolsContractsForConfigure(
-	require.GreaterOrEqual(t, len(evmChainSelectors), 1)
+	solChains := rt.Environment().BlockChains.SolanaChains()
+	require.GreaterOrEqual(t, len(solChains), 1)
+	solChain := slices.Collect(maps.Values(rt.Environment().BlockChains.SolanaChains()))[0]
```
```suggestion
	solChain := slices.Collect(maps.Values(solChains))[0]
```

graham-chainlink commented on deployment/ccip/changeset/v1_6_2/cs_configure_usdc_token_pool_test.go:
```diff
@@ -129,53 +125,41 @@ func setupUSDCTokenPoolsContractsForConfigure(
-	evmChain := env.BlockChains.EVMChains()[evmChainSelectors[0]]
+	evmChains := rt.Environment().BlockChains.EVMChains()
+	require.GreaterOrEqual(t, len(evmChains), 1)
+	evmChain := slices.Collect(maps.Values(rt.Environment().BlockChains.EVMChains()))[0]
```
```suggestion
	evmChain := slices.Collect(maps.Values(evmChains))[0]
```

graham-chainlink commented on deployment/ccip/changeset/cs_prerequisites_test.go:
```diff
@@ -3,52 +3,47 @@ package changeset_test
-	})
+	e, err := environment.New(t.Context(),
+		environment.WithEVMSimulatedN(t, 2),
+	)
```
i noticed sometimes we use WithLogger, sometimes we dont, is there a pattern?

jkongie replied:
No pattern. I've updated all the tests to include the logger now to retain existing behaviour, although I don't think logging in tests is actually necessary.

jkongie committed:
chore: update ccip tests to use new test engine.

This commit updates the ccip tests to use the new test engine. This
covers the simple conversions mainly revolving EVM and Aptos chains.

graham-chainlink reviewed (approved):
Massive effort here! Loooks good!
